diff --git a/bash-otube.desktop b/bash-otube.desktop
index 6bff784..a60913c 100755
--- a/bash-otube.desktop
+++ b/bash-otube.desktop
@@ -1,7 +1,7 @@
 #!/usr/bin/env xdg-open
 [Desktop Entry]
 Categories=AudioVideo;Player;Video;
-Comment=Play Online Videos in GUI Mode
+Comment=נגן סרטוני רשת במצב גרפי
 Exec=botube
 Icon=/usr/share/pixmaps/bash-otube.svg
 Name=Bash-OTube
diff --git a/bash-otube.sh b/bash-otube.sh
index 638d7ec..95f6151 100755
--- a/bash-otube.sh
+++ b/bash-otube.sh
@@ -64,17 +64,17 @@ else
 	if mplayer &> /dev/null ;then
 		PLAYER="mplayer" 
 		if youtube-dl --version &> /dev/null;then
-			echo "$debug Found youtube-dl"
+			echo "$debug נמצא youtube-dl"
 		else
 			zenity --error \
-			--title="YouTubeDL Not Found" \
-			--text="You-Tube Dl not found"
+			--title="YouTubeDL לא מותקן" \
+			--text="You-Tube Dl לא מותקן"
 			exit
 		fi
 	else
 		zenity --error \
-			--title="mplayer/mpv not found" \
-			--text="mplayer/mpv package not installed"
+			--title="mplayer/mpv לא מותקן" \
+			--text="mplayer/mpv לא מותקן"
 		exit
 	fi
 fi
@@ -86,9 +86,9 @@ fi
 bash_otube() {
 	url=`zenity --entry \
 	--title="Bash-OTube $version" \
-	--text="Video URL:" \
-	--ok-label="Play Video!" \
-	--cancel-label="Quit"` 
+	--text="כתובת הסרטון:" \
+	--ok-label="נגן סרטון!" \
+	--cancel-label="צא"` 
 	if [ $? -ne 0 ];then
 		echo "$debug Bye Bye!"
 		exit
@@ -100,11 +100,11 @@ bash_otube() {
 			echo "$debug Waiting for youtube-dl to start.. [5s]"
 			#DO NOT DELETE THE DELAY LINE!
 			sleep $delay | zenity --progress \
-			--title="Downloading Video Info.." \
+			--title="מוריד מידע אודות הסרטון.." \
 			--auto-close \
 			--pulsate \
 			--no-cancel \
-			--text "Downloading Video Info.." &
+			--text "מוריד מידע אודות הסרטון.." &
 			sleep $delay
 			VideoTitle=`youtube-dl --get-title $url`
 			$PLAYER -title "${VideoTitle}" "${OTubeVideo}.part"
diff --git a/bash-player.desktop b/bash-player.desktop
index e4ce9f6..a9f2869 100755
--- a/bash-player.desktop
+++ b/bash-player.desktop
@@ -1,7 +1,7 @@
 #!/usr/bin/env xdg-open
 [Desktop Entry]
 Categories=AudioVideo;Player;Video;
-Comment=A simple GUI for Mplayer
+Comment=ממשק גרפי פשוט ל - mplayer
 Exec=bmplayer
 Icon=/usr/share/pixmaps/bash-player.svg
 MimeType=audio/ac3;audio/mp4;audio/mpeg;audio/vnd.rn-realaudio;audio/vorbis;audio/x-adpcm;audio/x-matroska;audio/x-mp2;audio/x-mp3;audio/x-ms-wma;audio/x-vorbis;audio/x-wav;audio/mpegurl;audio/x-mpegurl;audio/x-pn-realaudio;audio/x-scpls;video/avi;video/mp4;video/flv;video/mpeg;video/quicktime;video/vnd.rn-realvideo;video/x-matroska;video/x-ms-asf;video/x-msvideo;video/x-ms-wmv;video/x-ogm;video/x-theora;video/webm;
diff --git a/bmplayer.sh b/bmplayer.sh
index c2c73aa..de5912e 100755
--- a/bmplayer.sh
+++ b/bmplayer.sh
@@ -129,19 +129,19 @@ cversions() {
 #help function
 help() {
 	zenity  --list \
-	--title="Mplayer - Shortcut" \
-	--ok-label="Return" \
-	--cancel-label="exit" \
+	--title="Mplayer - קיצורים" \
+	--ok-label="חזרה" \
+	--cancel-label="יציאה" \
 	--width=100 \
 	--height=400 \
-	--text "Help Options for Mplayer Type this with Your Keyboard in Mplayer" \
-	--column "Help" \
-	--column "Key" \
-	"Full Screen" f \
-	"Volume Up" 0 \
-	"Volume Down" 9 \
-	"Quit/Stop" "Esc" \
-	"Pause" "Space" \
+	--text "קיצורי מקשים לנגן הווידאו" \
+	--column "עזרה" \
+	--column "מקש" \
+	"מסך מלא" f \
+	"הגברת עוצמת השמע" 0 \
+	"הנמכת עוצמת השמע" 9 \
+	"יציאה/Stop" "ESC" \
+	"השהייה" "Space" \
 	"v" "Toggle subtitle visibility." \
 	"F" "Toggle displaying forced subtitles ."\
 	"T" "Toggle stay-on-top (also see -ontop)." \
@@ -180,8 +180,8 @@ bash_player() {
 		--width=100 \
 		--height=150 \
 		--title="Bash-player - $version" \
-		--ok-label="Select video" \
-		--cancel-label="Help"
+		--ok-label="בחירת סרטון" \
+		--cancel-label="עזרה"
 	else
 		echo "$debug No Startup Dialog."
 	fi
@@ -196,17 +196,17 @@ bash_player() {
 	else
 		#select video
 		video=`zenity --file-selection \
-			--title="Select a video" \
+			--title="בחירת סרטון" \
 			--filename=$folder`
 		if [ $? -ne 0 ]; then
 			echo "$debug no File selected."
 			exit
 		elif [ $subtitlesDialog -eq 1 ]; then
 			zenity --question \
-				--title="Add subtitles"  \
-				--text="Subtitles? \n\t(You can disable this Dialog)"  \
-				--ok-label="Yes, add" \
-				--cancel-label="No, play video"
+				--title="הוספת כתוביות"  \
+				--text="כתוביות? \n\t(אתה יכול להסתיר חלון זה)"  \
+				--ok-label="כן, הוסף" \
+				--cancel-label="לא, נגן סרטון"
 
 			if [ $? -ne 0 ]; then
 				$PLAYER -title "Bash-player $version" $mplayerpa "$video"
@@ -217,7 +217,7 @@ bash_player() {
 				fi
 			else
 				subtitles=`zenity --file-selection \
-				--title="Select subtitles" \
+				--title="בחירת כתוביות" \
 				--filename=$folder`
 
 				#run mplayer 
